app/models/responses/StatusResponse.scala

```diff
@@ -19,17 +19,19 @@ package models.responses
 import models.responses.ErrorResponse.StatusResponseError
 import org.slf4j.LoggerFactory
 import play.api.http.Status.OK
-import play.api.libs.json.{JsError, JsSuccess, Json, Reads}
+import play.api.libs.json.{JsError, JsSuccess, Json, Reads, Writes}
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
 
-case class StatusResponse(online: Boolean)
+case class StatusResponse(departuresWebHealthy: Boolean)
 
 object StatusResponse {
 
   private val logger = LoggerFactory.getLogger(classOf[StatusResponse])
 
   implicit val reads: Reads[StatusResponse] = Json.reads[StatusResponse]
 
+  implicit val writes: Writes[StatusResponse] = Json.writes[StatusResponse]
```

Comment:
Is it worth making these an OFormat since they're not doing anything custom?

Reply:
changed this to use oformat